packages/client/components/ErrorBoundary.tsx

```diff
@@ -25,21 +23,11 @@ class ErrorBoundary extends Component<Props, State> {
   }
 
   componentDidCatch(error: Error, errorInfo: ErrorInfo) {
-    const {atmosphere} = this.props
```

Comment:
By creating the `LogRocketManager` class and moving the Sentry user identification into `AnalyticsPage`, we no longer need to reach into the Atmosphere class to fetch this information when an error occurs.